hw/dv/tools/dvsim/questa.hjson

```diff
@@ -0,0 +1,106 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  build_cmd:  "vlib work && {job_prefix} vlog"
+  run_cmd:    "{job_prefix} vsim"
+
+  // Indicate the tool specific helper sources - these are copied over to the
+  // {tool_srcs_dir} before running the simulation.
+  tool_srcs:  ["{dv_root}/tools/questa/*"]
+
+  build_opts: ["-sv",
+               "-timescale 1ns/1ps",
+               "+incdir+/${QUESTA_HOME}/verilog_src/uvm-1.2/src/",
+               "/${QUESTA_HOME}/verilog_src/uvm-1.2/src/uvm_pkg.sv",
+               "-mfcu"
```

Comment:
please add comment for this too, not sure what it means.

Reply:
For -mfcu I hope, added